docs/rfcs/015-iiif-av-delivery-channel-settings.md

```diff
@@ -0,0 +1,132 @@
+# How are iiif-av settings going to be maintained in protagonist

```

Comment:
I think it's worth adding brief details around what we spoke about this morn (how they're handled, engine source of truth etc).